app/vmselect/promql/transform.go

```diff
@@ -826,6 +826,9 @@ func transformHistogramQuantiles(tfa *transformFuncArg) ([]*timeseries, error) {
 	// Calculate quantile individually per each phi.
 	var rvs []*timeseries
 	for _, phiArg := range phiArgs {
+		if len(phiArg) == 0 || (phiArg[0].Values[0] < 0 || phiArg[0].Values[0] > 1) {
```

Comment:
weird why in first place `Values` are nil. Do you think it is expected?

Reply:
The main problem here is that user can set invalid params, we expected number, but user can set for example `phi0.3`, which will parse to `metricsql.MetricExpr` instead of `metricsql.NumberExpr`. In that case phiArg will have 
`[]*promql.timeseries(nil)` and app is going to panic.
Also i will check metricsql library, maybe there would be a better place for fix